src/components/AddressSearch/index.tsx

```diff
@@ -189,18 +82,18 @@ function AddressSearch({
         [preferredLocale, resultTypes, isLimitedToUSA, locationBias],
     );
     const shouldShowCurrentLocationButton = canUseCurrentLocation && searchValue.trim().length === 0 && isFocused;
-    const saveLocationDetails = (autocompleteData, details) => {
-        const addressComponents = details.address_components;
+    const saveLocationDetails = (autocompleteData: GooglePlaceData, details: GooglePlaceDetail | null) => {
+        const addressComponents = details?.address_components;
         if (!addressComponents) {
             // When there are details, but no address_components, this indicates that some predefined options have been passed
             // to this component which don't match the usual properties coming from auto-complete. In that case, only a limited
             // amount of data massaging needs to happen for what the parent expects to get from this function.
-            if (_.size(details)) {
-                onPress({
-                    address: autocompleteData.description || lodashGet(details, 'description', ''),
-                    lat: lodashGet(details, 'geometry.location.lat', 0),
-                    lng: lodashGet(details, 'geometry.location.lng', 0),
```

Comment:
Are you sure that default values are not necessary anymore?

Reply:
Added